library/std/Cargo.toml

```
@@ -99,3 +100,13 @@ heap_size = 0x8000000
 name = "stdbenches"
 path = "benches/lib.rs"
 test = true
+
+[lints.rust.unexpected_cfgs]
+level = "warn"
+check-cfg = [
```

```suggestion
check-cfg = [
    'cfg(feature, values(any()))',
```

to match https://github.com/rust-lang/rust/blob/e8ada6ab253b510ac88edda131021d9878f2984f/src/bootstrap/src/core/builder.rs#L1553-L1561

(to be done for std, alloc and core)